packages/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -120,6 +120,7 @@ class AccordionTitle extends UIComponent<WithAsProp<AccordionTitleProps>, any> {
           {...applyAccessibilityKeyHandlers(accessibility.keyHandlers.content, unhandledProps)}
           start={Icon.create(indicatorWithDefaults, {
             defaultProps: {
+              name: 'stardust-circle',
```

Comment:
Same thing about all `Icon.create()`, we don't need to specify it under `defaultProps`